$OpenBSD: patch-src_celengine_parseobject_cpp,v 1.1 2019/06/15 18:47:47 cwen Exp $

ports-gcc fix for:
parseobject.cpp:280:10: error: converting to 'bool' from 'std::nullptr_t'
requires direct-initialization 

Index: src/celengine/parseobject.cpp
--- src/celengine/parseobject.cpp.orig
+++ src/celengine/parseobject.cpp
@@ -277,7 +277,7 @@ ParseStringList(Hash* table,
 {
 	Value* v = table->getValue(propertyName);
 	if (v == NULL)
-		return NULL;
+		return false;
 
 	// Check for a single string first.
 	if (v->getType() == Value::StringType)
